resources/install.sh

```diff
@@ -20,7 +20,7 @@ password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -
 sed -i s/MTIxMzQyNDMyZHdkY2RldmdyZWc=/$password/g /var/lib/gravity/resources/secrets.yaml
 
 /opt/bin/kubectl apply -f /var/lib/gravity/resources/namespace.yaml
-for filename in security secrets influxdb-secret smtp influxdb grafana heapster kapacitor telegraf alerts
+for filename in security secrets influxdb-secret smtp influxdb grafana heapster kapacitor rollups telegraf alerts
```

Comment:
How were the default rollups being created before?

Reply:
I moved it from grafana.yaml to dedicated file